pkg/aws/ec2/mock/mock_test.go

```diff
@@ -9,71 +9,62 @@ import (
 	"net"
 	"testing"
 
-	check "github.com/cilium/checkmate"
+	"github.com/stretchr/testify/require"
 
 	"github.com/cilium/cilium/pkg/aws/types"
-	"github.com/cilium/cilium/pkg/checker"
 	"github.com/cilium/cilium/pkg/ip"
 	"github.com/cilium/cilium/pkg/ipam/cidrset"
 	ipamTypes "github.com/cilium/cilium/pkg/ipam/types"
 )
 
-func Test(t *testing.T) {
-	check.TestingT(t)
-}
-
-type MockSuite struct{}
-
-var _ = check.Suite(&MockSuite{})
-
-func (e *MockSuite) TestMock(c *check.C) {
+func TestMock(t *testing.T) {
 	api := NewAPI([]*ipamTypes.Subnet{{ID: "s-1", AvailableAddresses: 100}}, []*ipamTypes.VirtualNetwork{{ID: "v-1"}}, []*types.SecurityGroup{{ID: "sg-1"}})
-	c.Assert(api, check.Not(check.IsNil))
+	require.NotNil(t, api)
 
 	eniID1, _, err := api.CreateNetworkInterface(context.TODO(), 8, "s-1", "desc", []string{"sg1", "sg2"}, false)
-	c.Assert(err, check.IsNil)
+	require.NoError(t, err)
 
 	eniID2, _, err := api.CreateNetworkInterface(context.TODO(), 8, "s-1", "desc", []string{"sg1", "sg2"}, false)
-	c.Assert(err, check.IsNil)
+	require.NoError(t, err)
 
 	_, err = api.AttachNetworkInterface(context.TODO(), 0, "i-1", eniID1)
-	c.Assert(err, check.IsNil)
+	require.NoError(t, err)
 
 	_, ok := api.enis["i-1"][eniID1]
-	c.Assert(ok, check.Equals, true)
+	require.True(t, ok)
```

Comment:
nit: the message before didn't have a comment either, but it's really useful to add one here if you can — or else you get error messages like "test failed: false was not true".

Reply:
make sense, however, this might be require more effort for this migration, as I run sed commands to change to require statements. So let's keep the same as what it is now.